modules/host/persist.go

```diff
@@ -3,39 +3,42 @@ package host
 import (
 	"path/filepath"
 
-	"github.com/NebulousLabs/Sia/encoding"
 	"github.com/NebulousLabs/Sia/modules"
+	"github.com/NebulousLabs/Sia/persist"
 )
 
+var persistMetadata = persist.Metadata{
+	Header:  "Sia Host",
+	Version: "0.3.3",
```

Comment:
yeah, that was my reasoning. We could do better in that respect though. To my knowledge, there are still two places where we don't use the same version as Sia: in the gateway connection protocol (version exchange) and in the shared file header. We should consider updating one or both of those.